docs/src/tricks.md

```diff
@@ -31,7 +31,7 @@ Similarly, you can use a template image and repaint it for the current theme.
 { pkgs, ... }:
 
 let
-  theme = "${pkgs.base16-schemes}/share/themes/catppuccin.yaml";
+  theme = "${pkgs.base16-schemes}/share/themes/catppuccin-latte.yaml";
```

Comment:
Can you elaborate on why the documentation should not use `catppuccin.yaml`:

```suggestion
  theme = "${pkgs.base16-schemes}/share/themes/catppuccin.yaml";
```

Reply:
When I tested the example the file wasn't there, replaced with 4 variant themes. `catppuccin.yaml` reference is stale, at least for me.

Reply:
Yes, the four variants are as follows:

- Latte
- Frappe
- Macchiato
- Mocha

It's not particularly important what theme this refers to, since it's only an example. But it should at least be a file which exists in case someone copies and pastes it.